build_artifacts/v0/v0.9/v0.9.0/cpu.env.in

```diff
@@ -9,6 +9,7 @@ conda-forge::pandas[version='>=2.0.3,<3.0.0']
 conda-forge::scikit-learn[version='>=1.3.1,<2.0.0']
 conda-forge::jinja2[version='>=3.1.2,<4.0.0']
 conda-forge::matplotlib[version='>=3.7.3,<4.0.0']
+conda-forge::sagemaker-headless-execution-driver[version'>=0.0.9,']
```

Comment:
Please remove trailing comma. Should be just `[version'>=0.0.9']`

Reply:
ack